src/dotnet/APIView/APIViewWeb/Models/ReviewGenPipelineParamModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace APIViewWeb.Models
+{
+    public class ReviewGenPipelineParamModel
+    {
+        public string ReviewID { get; set; }

```

Comment:
nit: Should be Id rather than ID for all properties.

Reply:
This might require change in pipeline side as well which is already in place, I will check If i can change it over there as well. If not, I will submit a separate PR to take care of this in both places.

Reply:
not a big deal if that is the case.